src/net/http.rs

```diff
@@ -93,6 +93,7 @@ use std::time::SystemTime;
 
 lazy_static! {
     static ref PATH_GETINFO : Regex = Regex::new(r#"^/v2/info$"#).unwrap();
+    static ref PATH_GETPOXINFO : Regex = Regex::new(r#"^/v2/pox$"#).unwrap();
```

Comment:
Why is this a separate API call?  Shouldn't this just be part of `/v2/info`, since `/v2/info` _already reports on PoX state_?

Reply:
I initially started with a new node in `v2/info` and ended up creating a dedicated endpoint.
The schema is pretty simple right now, but will might become busier in the future (recipients selected for current cycle, history of last N reward cycles, etc), we probably want to avoid introducing noise in `v2/indo` and keep it light.